ci/scripts/shipit

```diff
@@ -72,7 +72,7 @@ bosh -n create-release "releases/${RELEASE_NAME}/${RELEASE_NAME}-${VERSION}.yml"
 cd -
 
 # SC2155 discourages variable assignment and export in the same line.
-RELEASE_TGZ=${REPO_ROOT}/releases/${RELEASE_NAME/}${RELEASE_NAME}-${VERSION}.tgz
+RELEASE_TGZ=${REPO_ROOT}/releases/${RELEASE_NAME}/${RELEASE_NAME}-${VERSION}.tgz
```

Comment:
We had also overlooked this incorrectly wrapped variable when doing the shellcheck adaptations / cleanup.